### DISCUSSION TITLE
Unable to Build the Gerber Files

### DISCUSSION THREAD

**Discussion by pokun** [sanni/cartreader#115] [Need help]
I'm trying to build the gerber files from the Eagle PCB source files. I'm using Eagle 7.6.0 and successfully followed the instructions in the readme up to the point of creating the DNS file. But when executing the "FreeRouting1.jar" file I get this error:
"Error: Could not find or load main class [path to FreeRouting1.jar]"

I'm unsure what to do, and if all files are in the correct place or if there is some other problem. "FreeRouting1.jar" is in the "ulp" directory.

**Comment by sanni** [comment]
It works with Windows 10 x64 and the jre-8u281-windows-x64.exe installed from https://www.java.com/de/download/manual.jsp.
FreeRouting1.jar can be in any folder. It might not work with newer Java versions.

**Reply by pokun** [reply]
I installed "jre-8u281-windows-x64.exe" and tried opening "FreeRouting1.jar" with it. I get the exact same error again. Has this been tested on any other computer than it was built on?

**Comment by sanni** [comment]
You can try a newer version of freerouting that uses Java 11: https://github.com/freerouting/freerouting/releases/tag/v1.4.4

**Reply by pokun** [reply]
Thank you that version of freerouting did seem to work. I let it route for a few hours and then proceeded to execute the exported script in Eagle. When I did that however, there were some errors saying that the light version of Eagle couldn't do some things as it was outside the limited area. I clicked OK on all errors and proceeded anyway. I could build gerber files, but they where very different from the pre-built ones on Github. Is the professional version of Eagle required or did I do something else wrong?

It doesn't matter so much as I figured out that since I wasn't going to change the copper anyway I could skip the ripup and the whole autorouting process, and do the CAM processing directly (things like silk-screen changes can be made without having to ripup and reroute). I was able to build the Mark III adapter among other things that were only included as Eagle design files on Github. Is there a reason these adapters are not included as gerber files on Github?
